README.md

```diff
@@ -1,2 +1,2 @@
 # rpp2205-sijia-service
-This is a service created by Sijia Tao to provide database API to Atelier project
\ No newline at end of file
+This is a service created by Sijia Tao to provide database API to Atelier project:wq
```

Comment:
I'm assuming the :wq at the end was an accident.  Not a blocker for merge, but flagging it here.